src/ios/SimpleCameraPreview.m

```diff
@@ -11,6 +11,23 @@ @implementation SimpleCameraPreview
 
 BOOL torchActivated = false;
 
+- (void) getRatio:(CDVInvokedUrlCommand*)command {
+    NSDictionary* config = command.arguments[0];
+    @try {
+        if (config[@"targetSize"] != [NSNull null]) {
```

Comment:
here in objective-c, no need to check the "null" of type string, like you did in the java section ?